lib/utilities.c

```diff
@@ -1216,8 +1212,8 @@ RmOff rm_offset_get_from_path(_UNUSED const char *path, _UNUSED RmOff file_offse
     return 0;
 }
 
-int rm_offsets_match(char *path1, char *path2) {
-    return (path1 == path2);
+RmOffsetsMatchCode rm_offsets_match(char *path1, char *path2) {
+    return (path1 == path2) ? RM_OFFSETS_MATCH : RM_OFFSETS_NO_DATA;
```

Comment:
[embarrassing](https://github.com/sahib/rmlint/commit/cf89f219eb5de0fd74104848a48924d0652f1fdb) legacy of my non-c background

Will fix... and maybe rename `rm_offsets_match` to `rm_link_type(path1, path2)` which will help avoid confusion